.github/workflows/test-build-deploy.yml

```diff
@@ -228,10 +240,10 @@ jobs:
     if: (startsWith(github.ref, 'refs/tags/') || startsWith(github.ref, 'refs/heads/r') ) && github.event_name == 'push' && github.repository == 'grafana/mimir'
     runs-on: ubuntu-latest
     container:
-      image: grafana/mimir-build-image:pr3976-e7cae18e3
+      image: grafana/mimir-build-image:goupdate-751733fe1
     steps:
       - name: Checkout repository
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
```

Comment:
Do we need to run `run: git config --global --add safe.directory /__w/mimir/mimir` here too?

Reply:
Since it doesn't fail for some reason, I would say it's not necessary?